### PR TITLE
feat(widget): New component text

### PR DESCRIPTION
**Pull request by MarvinFlm**
# Description

Add new component txt to the library of widget

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the needed labels
- [x] I have linked this PR to a clickup task
- [x] I have tested this code
- [ ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

### CODE REVIEW

File: .flutter-plugins-dependencies
Comment (lucas-louis) [inline .flutter-plugins-dependencies:1]: I don't think it needs to be pushed

File: .flutter-plugins
Comment (lucas-louis) [inline .flutter-plugins:1]: I don't think it needs to be pushed

File: lib/src/text.dart [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        fontSize = 64;
+        break;
+      case TxtStyles.xxxxxxxxl:
+        fontweight = FontWeight.bold;
+        fontSize = 72;
+        break;
+      default:
+        fontweight = FontWeight.normal;
+        fontSize = 12;
+        break;
```
Comment (lucas-louis) [inline lib/src/text.dart]: For the fontWeight use the numeric value instead

**Review by lucas-louis — CHANGES_REQUESTED** [review]

File: lib/src/text.dart (hunk 1 above)
Comment (MarvinFlm) [reply lib/src/text.dart]: Et mettre a jour la version du package

File: .flutter-plugins
Comment (lucas-louis) [inline .flutter-plugins:1]: ...

File: .flutter-plugins-dependencies
Comment (lucas-louis) [inline .flutter-plugins-dependencies:1]: ...

**Review by lucas-louis — CHANGES_REQUESTED** [review]
Useless files

**Review by lucas-louis — APPROVED** [review]
LGTM

**Pull request merged**